responses/__init__.py

```diff
@@ -565,6 +565,13 @@ def add(
 
         if adding_headers is not None:
             kwargs.setdefault("headers", adding_headers)
+        if "content_type" in kwargs and "headers" in kwargs:
+            header_keys = list(map(lambda x: x.lower(), kwargs["headers"].keys()))
```

Comment:
could this be simplified to?
```suggestion
            header_keys = [header.lower() for header in kwargs["headers"]]
```